yatulearn/yatulearn
# Added USER AUTHENTICATION using django

### **Added user authentication functionalities using Django.**

>> Users can now sign up and log in through the application.
>> Once logged in, users have the option to log out from the application, which is accessible at the bottom of the page.

**How to Test**
1. Install Django: Ensure Django is installed in your environment. 
   `pip install django`

2. Navigate to the loginSignup folder:
   Change to the` loginSignup` directory where the `manage.py` file is located.
  **cd loginSignup**

3. Run the server:
   `python manage.py runserver`

4. Open the application:
    Access the application in your web browser at `http://127.0.0.1:8000/.`

5. Testing Authentication:
   Sign up with a new user.
   Log in with the created user credentials.
  Log out using the logout link at the bottom of the page.

**Screenshots**
login page
![login_page](https://github.com/user-attachments/assets/2a6e7d93-d018-4825-bdc2-b8cb4ff83c84)
Signup page
![signup](https://github.com/user-attachments/assets/c839fbb9-7ea5-4e6a-a51a-868f16ba4e42)
